draft-ietf-quic-transport.md

```diff
@@ -4395,13 +4439,22 @@ A client MUST NOT reset the packet number for any packet number space after
 processing a Retry packet; {{packet-0rtt}} contains more information on this.
 
 A server acknowledges the use of a Retry packet for a connection using the
-original_connection_id transport parameter (see
-{{transport-parameter-definitions}}).  If the server sends a Retry packet, it
-MUST include the Destination Connection ID field from the client's first
-Initial packet in the transport parameter.
+retry_connection_id transport parameter; see
+{{transport-parameter-definitions}}.  If it sends a Retry packet, the server
+also subsequently includes the value of the Source Connection ID field from
+the Retry packet in its retry_connection_id transport parameter.
+
+A server always includes an original_connection_id transport parameter. If it
```

Comment:
I think we can remove this entire paragraph as it appears to be redundant with other parts of this PR, and it seems to have discrepancies.